p2p/src/main/java/bisq/network/p2p/network/Connection.java

```diff
@@ -724,7 +725,8 @@ public void run() {
 
                     NetworkEnvelope networkEnvelope = networkProtoResolver.fromProto(proto);
                     lastReadTimeStamp = now;
-                    log.debug("<< Received networkEnvelope of type: " + networkEnvelope.getClass().getSimpleName());
+                    if (log.isDebugEnabled())
+                        log.debug("<< Received networkEnvelope of type: " + networkEnvelope.getClass().getSimpleName());
 
```

Comment:
I would instead suggest:

```suggestion
log.debug("<< Received networkEnvelope of type: {}", networkEnvelope.getClass().getSimpleName());
```

Reply:
ok that will still do the `networkEnvelope.getClass().getSimpleName()` call but that's probably not expensive.

Reply:
As far I understand it does it only at debug log level

Reply:
It looks like it will have to do that call to `getSimpleName()` but not build the string at debug log level. I think that's an acceptable balance so I change my opinion and would go with what @freimair suggests.

Reply:
i already changed it to his suggestion